src/components/sbb-image/sbb-image.tsx

```diff
@@ -224,14 +221,22 @@ export class SbbImage {
   @Prop() public pictureSizesConfig?: string;
 
   /**
-   * Based on the variant, we apply specific aspect ratios
-   * to the image accross all viewports.
+   * Variant Teaser-Hero: we apply specific aspect ratios to the image accross all viewports.
```

Comment:
```suggestion
   * Variant Teaser-Hero: we apply specific aspect ratios to the image across all viewports.
```

Reply:
Here you write across all viewports, in css I see that it is starting only from small (zero to small would ned be overwritten). Or am I wrong?